keps/sig-node/4800-cpumanager-split-uncorecache/README.md

```diff
@@ -512,7 +512,7 @@ N/A
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-N/A
+N/A, because the feature is self-contianed and have no dependencies outside the kubelet. 
```

Comment:
i think it's not applicable, but not because it's self-contained, but because an upgrade->downgrade->upgrade would be handled similarly as a reboot. the cpu manager state file would be removed and pods would need to be drained to be realigned on uncore cache lines in both situations.

Reply:
I agree with this, @ffromani do you?

Reply:
Yes, seems fair and in the end we're converging on the same topic from different perspectives, so I agree

Reply:
Okay I added it, lmk if it's good